command/polybftmanifest/manifest_init.go

```diff
@@ -0,0 +1,211 @@
+package polybftmanifest
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/genesis"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	manifestPathFlag      = "path"
+	premineValidatorsFlag = "premine-validators"
+	validatorsFlag        = "validators"
+	validatorsPathFlag    = "validators-path"
+	validatorsPrefixFlag  = "validators-prefix"
+
+	defaultValidatorPrefixPath = "test-chain-"
+	defaultManifestPath        = "./manifest.json"
+
+	nodeIDLength       = 53
+	ecdsaAddressLength = 42
+	blsKeyLength       = 2
+)
+
+var (
+	params = &manifestInitParams{}
+)
+
+// GetCommand returns the rootchain emit command
+func GetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "polybft-manifest",
+		Short:   "Initializes manifest file",
+		PreRunE: runPreRun,
+		Run:     runCommand,
+	}
+
+	setFlags(cmd)
+
+	return cmd
+}
+
+func runPreRun(_ *cobra.Command, _ []string) error {
+	return params.validateFlags()
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.manifestPath,
+		manifestPathFlag,
+		defaultManifestPath,
+		"the file path where manifest file is going to be stored",
+	)
+
+	cmd.Flags().StringVar(
+		&params.validatorsPath,
+		validatorsPathFlag,
+		"./",
+		"root path containing polybft validators private keys",
+	)
+
+	cmd.Flags().StringVar(
+		&params.validatorsPrefixPath,
+		validatorsPrefixFlag,
+		defaultValidatorPrefixPath,
+		"folder prefix names for polybft validator private keys",
+	)
```

Comment:
If this need to read the private keys, it should be compatible with secrets manager, should be capable of reading secrets from the `secretsManagerConfig.json`

Reply:
Maybe comment is imprecise as we don't need private keys per se, but are deducing ECDSA address, BLS public key and p2p node id. So this works exactly the same as polybft genesis population did: either provide path and prefixes in order to read validator information from the local storage, or provide such information using `--validators` flag. This is not reading single validator private key, but all the genesis validators information. 
Can you give an example of code snippet in IBFT genesis command which consults secrets manager in order to extract such information, so that I get more context?

Reply:
Makes sense, then the flag help message is not totally correct.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1005/commits/f84bf34abe858c448eaa998b6e389c301ba58708